### ISSUE TITLE
migtissera/Synthia-70B-v1.2b · Hugging Face

### ISSUE THREAD

**Issue by irthomasthomas** [irthomasthomas/undecidability#593]
- [ ] [migtissera/Synthia-70B-v1.2b · Hugging Face](https://huggingface.co/migtissera/Synthia-70B-v1.2b)

# migtissera/Synthia-70B-v1.2b · Hugging Face

**DESCRIPTION:** Change from 1.2 -> 1.2b: More data, 14 days of training for 1 epoch.

All Synthia models are uncensored. Please use it with caution and with best intentions. You are responsible for how you use Synthia.

To evoke generalized Tree of Thought + Chain of Thought reasoning, you may use the following system message:

Elaborate on the topic using a Tree of Thoughts and backtrack when necessary to construct a clear, cohesive Chain of Thought reasoning. Always answer without hesitation.

**Synthia-70B-v1.2b**

SynthIA (Synthetic Intelligent Agent) is a LLama-2-70B model trained on Orca style datasets. It has been fine-tuned for instruction following as well as having long-form conversations.

**License Disclaimer:**

This model is bound by the license & usage restrictions of the original Llama-2 model, and comes with no warranty or guarantees of any kind.

**Evaluation**

We evaluated Synthia-70B-v1.2b on a wide range of tasks using Language Model Evaluation Harness from EleutherAI.

Here are the results on metrics used by HuggingFaceH4 Open LLM Leaderboard

| Task          | Metric    | Value |
|---------------|-----------|-------|
| arc_challenge | acc_norm  | 68.77 |
| hellaswag     | acc_norm  | 87.57 |
| mmlu         | acc_norm  | 68.81 |
| truthfulqa_mc | mc2       | 57.69 |
| Total Average | -         | 70.71 |

**Example Usage**

Here is prompt format:

SYSTEM: Elaborate on the topic using a Tree of Thoughts and backtrack when necessary to construct a clear, cohesive Chain of Thought reasoning. Always answer without hesitation.

USER: How is a rocket launched from the surface of the earth to Low Earth Orbit?

ASSISTANT:

[Link to Synthia-70B-v1.2b](https://huggingface.co/migtissera/Synthia-70B-v1.2b)

####

**Comment by irthomasthomas** [comment]
### Related issues

### #459: llama2
<details><summary>### Details</summary>Similarity score: 0.88
- [ ] [llama2](https://ollama.ai/library/llama2)

Llama 2
========

The most popular model for general use.

*265.8K Pulls*
*Updated 4 weeks ago*

Overview
--------

Llama 2 is released by Meta Platforms, Inc. This model is trained on 2 trillion tokens, and by default supports a context length of 4096. Llama 2 Chat models are fine-tuned on over 1 million human annotations, and are made for chat.

CLI
---

Open the terminal and run

```bash
ollama run llama2
```

API
---

Example using curl:

```bash
curl -X POST http://localhost:11434/api/generate -d '{
  "model": "llama2",
  "prompt":"Why is the sky blue?"
 }'
```

API documentation
-----------------

Memory requirements
-------------------

-  7b models generally require at least 8GB of RAM
-  13b models generally require at least 16GB of RAM
-  70b models generally require at least 64GB of RAM

If you run into issues with higher quantization levels, try using the q4 model or shut down any other programs that are using a lot of memory.

Model variants
--------------

-   **Chat**: fine-tuned for chat/dialogue use cases. These are the default in Ollama, and for models tagged with `-chat` in the tags tab.

    Example: `ollama run llama2`

-   **Pre-trained**: without the chat fine-tuning. This is tagged as `-text` in the tags tab.

    Example: `ollama run llama2:text`

By default, Ollama uses 4-bit quantization. To try other quantization levels, please use the other tags. The number after the `q` represents the number of bits used for quantization (i.e. `q4` means 4-bit quantization). The higher the number, the more accurate the model is, but the slower it runs, and the more memory it requires.

References
----------

-   [Llama 2: Open Foundation and Fine-Tuned Chat Models](https://metastring.com/llama2)
-   [Meta’s Hugging Face repo](https://huggingface.co/Meta)

#### Suggested labels
#### {   "label-name": "llama2-model",   "description": "A powerful text model for chat, dialogue, and general use.",   "repo": "ollama.ai/library/llama2",   "confidence": 91.74 }</details>


### #552: LargeWorldModel/LWM-Text-Chat-1M · Hugging Face
<details><summary>### Details</summary>Similarity score: 0.87
- [ ] [LargeWorldModel/LWM-Text-Chat-1M · Hugging Face](https://huggingface.co/LargeWorldModel/LWM-Text-Chat-1M)


# LargeWorldModel/LWM-Text-Chat-1M · Hugging Face

**DESCRIPTION:**

LWM-Text-1M-Chat Model Card

**Model details**

Model type: LWM-Text-1M-Chat is an open-source model trained from LLaMA-2 on a subset of Books3 filtered data. It is an auto-regressive language model, based on the transformer architecture.

Model date: LWM-Text-1M-Chat was trained in December 2023.

Paper or resources for more information: [https://largeworldmodel.github.io/](https://largeworldmodel.github.io/)

**URL:** [https://huggingface.co/LargeWorldModel/LWM-Text-Chat-1M](https://huggingface.co/LargeWorldModel/LWM-Text-Chat-1M)

#### Suggested labels
#### {'label-name': 'Open-source Models', 'label-description': 'Models that are publicly available and open-source for usage and exploration.', 'gh-repo': 'huggingfaceco/LargeWorldModel/LWM-Text-Chat-1M', 'confidence': 56.11}</details>


### #489: Hallucinations Leaderboard - a Hugging Face Space by hallucinations-leaderboard
<details><summary>### Details</summary>Similarity score: 0.86
- [ ] [Hallucinations Leaderboard - a Hugging Face Space by hallucinations-leaderboard](https://huggingface.co/spaces/hallucinations-leaderboard/leaderboard)

 # Hallucinations Leaderboard - a Hugging Face Space by hallucinations-leaderboard

## Description

[Hugging Face Space](https://huggingface.co/spaces/hallucinations-leaderboard/leaderboard)

Explore the Hallucinations Leaderboard, a space dedicated to tracking and benchmarking the performance of language models in generating accurate and coherent responses. This leaderboard focuses on the potential issue of hallucinations, where models generate plausible but factually incorrect information.

By highlighting the best models and fostering competition, we aim to encourage developers to create more reliable and truthful language generation systems. This space is a continuously updated resource for researchers, developers, and enthusiasts interested in the evaluation and improvement of language model performance.

![Hallucinations Leaderboard Screenshot](https://i.imgur.com/wgD8NrN.png)

## Key Features

- **Regular Updates:** The leaderboard is frequently updated with the latest model performances, ensuring that you always have access to the most recent data.

- **Easy Comparison:** Compare the performance of various models side-by-side to identify trends and determine which models perform best in specific tasks or scenarios.

- **Benchmarking Tools:** Utilize built-in benchmarking tools to test your own models and see how they stack up against the competition.

- **Community Engagement:** Join a community of developers, researchers, and enthusiasts dedicated to improving language model performance and reducing hallucinations.

## Explore the Hallucinations Leaderboard

Visit the [Hallucinations Leaderboard Hugging Face Space](https://huggingface.co/spaces/hallucinations-leaderboard/leaderboard) to start exploring and stay up-to-date with the latest advancements in language model performance and hallucination reduction.

**Note:** This leaderboard focuses on English language models. If you would like to contribute data for other languages, please contact the maintainers.

#### Suggested labels
#### {   "label-name": "hugging-face-space",   "description": "A dedicated space for sharing models, datasets, and other resources within the Hugging Face community.",   "repo": "hallucinations-leaderboard",   "confidence": 91.35 }</details>


### #158: TinyLlama-1.1B-intermediate-step-1195k-token-2.5T-exl2 · Hugging Face
<details><summary>### Details</summary>Similarity score: 0.86
- [ ] [zakoman/TinyLlama-1.1B-intermediate-step-1195k-token-2.5T-exl2 · Hugging Face](https://huggingface.co/zakoman/TinyLlama-1.1B-intermediate-step-1195k-token-2.5T-exl2)

    8bpw exllamav2 quantisation of TinyLlama/TinyLlama-1.1B-intermediate-step-1195k-token-2.5T

Calibration dataset used is vicgalle/alpaca-gpt4

Original model card

TinyLlama-1.1B

https://github.com/jzhang38/TinyLlama

The TinyLlama project aims to pretrain a 1.1B Llama model on 3 trillion tokens. With some proper optimization, we can achieve this within a span of "just" 90 days using 16 A100-40G GPUs 🚀🚀. The training has started on 2023-09-01.</details>
